arches/management/commands/updateproject.py

```diff
@@ -19,216 +15,27 @@ def handle(self, *args, **options):
         answer = input(
             "This operation will upgrade your project to version 8.0\n"
             "This will replace the following files in your project:\n"
-            "  - <project>/apps.py\n"
-            "  - .github/actions/build-and-test-branch/action.yml\n"
-            "  - .github/workflows/main.yml\n"
-            "  - tsconfig.json\n"
-            "  - vitest.config.mts\n"
-            "  - webpack/webpack-utils/build-filepath-lookup.js\n"
-            "  - webpack/webpack.common.js\n"
-            "  - webpack/webpack.config.dev.js\n"
-            "  - webpack/webpack.config.prod.js\n"
+            "  - eslint.config.mjs\n"
             "Continue? "
         )
 
         if answer.lower() in ["y", "yes"]:
-            self.update_to_v8()
+            self.update_to_v8_1()
         else:
             self.stdout.write("Operation aborted.")
 
-    def update_to_v8(self):
```

Comment:
The 7.x version of this command left all 7.x versions in this file until 8 hit. Should we keep to that?